app/javascript/components/components/Header.js

```diff
@@ -14,8 +16,9 @@ class Header extends Component {
     console.log("current_user:", current_user)
     return (
       <>
-        <h1>React in Rails with Devise</h1>
+        <h1>nest.</h1>
         <Nav>
+          <NavItem><Link src={Home} className="nav-link">Home</Link></NavItem>
```

Comment:
Does this link work? I feel like you would want to use a NavLink with a `to` attribute.

Reply:
have not been able to try, ill be trying it once i create a link to the apartments for next PR